orlandoc01/io-flow-types
# v0.2 release (WIP)

## 0.2.0 (c1694d5)
### Added
- Add new `ValidationError` class with default error message using `PathReporter` logic
- Add new `AggregateError` class for error handling multiple `ValidationError` instances
### Changed
- `Validation<A>` is now `Either<AggregateError, A>`
    * `AggregateError` is subclass of `Array<ValidationError>`, so should be backwards compatible
- Add `assert` method on `Type` class
- Add `getAssert` method on `Type` class
- `toString` will now render the name for function or a default string with its arity
- Restructured tests directory
- Add optional `message` arg to `failure`
### Removed
- remove `PathReporter` - `ValidationError` message uses path logic for message
- remove `ThrowReporter` - added `assert` method mimicks the logic
- remove `getFunctionName` export
- remove `getContextEntry` export
- remove `getValidationError` export
